### ISSUE TITLE
Training code for custom dataset

### ISSUE THREAD

**Issue by cvmlarun** [RyersonMultimediaLab/DynaSeg#1]
Training code for custom dataset

**Comment by RyersonMultimediaLab** [comment]
The script DynaSeg_COCO.py is built following MMDetection standards and is primarily used to generate a confusion matrix (accuracy, mIoU, and mAcc) for all classes, including "thing" and "stuff" categories, similar to DenseSiam and PiCIE.

However, if you are working with a custom dataset, it is recommended to use DynaSeg_SCF_BSD_VOC.py. This script can run independently, and you only need to specify the path to your dataset for it to function properly.

**Comment by inna-kucher** [comment]
I would like to take pre-trained resnet-18 , couple it with FPN and train on a custom dataset. I don't think 
 DynaSeg_SCF_BSD_VOC.py would let me do it. 

I cannot understand how the cluster labels are estimated, since the method to find the optimal clusters is not applied in this script ... Could you please let me know how the cluster labels are estimated ?

Thank you
